functions/filter_data.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Helper function that takes in the currently loaded data and the filters object and returns a new
+ * data objected that has been filtered.
+ * @param {obj} data // Coming from state.data
+ * @param {obj} filters // Coming from state.filters
+ */
+export default function filterData(records, filters) {
```

Comment:
I extracted this function from `pages/data.js` so we could more easily test it in isolation without loading up all of the other things that the "Explore the Data" page needs. I think it would be valuable to have integration tests around what the user actually sees on the page at some point, but I wanted to start simple.